packages/bpk-tokens/src/react-native-output-test.js

```diff
@@ -23,7 +23,7 @@ import _ from 'lodash';
 import androidTokens from '../tokens/android/base.raw.json';
 import iosTokens from '../tokens/ios/base.raw.json';
 
-const ANDROID_ONLY_PROPS = [];
+const ANDROID_ONLY_PROPS = ['FONT_FAMILY_MEDIUM'];
```

Comment:
What is better... 

**Baking conditions into tokens:**
_ios:_
```
FONT_FAMILY: 'System'
FONT_FAMILY_MEDIUM: 'System'
```
_android:_
```
FONT_FAMILY: 'sans-serif'
FONT_FAMILY_MEDIUM: 'sans-serif-medium'
```
_component:_
```
style: {
  fontFamily: emphasized ? TOKENS.FONT_FAMILY : TOKENS.FONT_FAMILY_MEDIUM
}
```

Or, **having conditional tokens and adding extra conditions into components:**
_component:_
```
style: {
  fontFamily: Platform.OS === 'android' ? (emphasized ? TOKENS.FONT_FAMILY : TOKENS.FONT_FAMILY_MEDIUM)  : TOKENS.FONT_FAMILY
}
```

Reply:
I like making it explict as this is a techincal implementation detail of Android and thus the code should relfect that we have a special case here.

Reply:
react native utils? :trollface: